packages/dataviews/src/dataviews-layouts/table/column-header-menu.tsx

```diff
@@ -102,151 +102,160 @@ const _HeaderMenu = forwardRef( function HeaderMenu< Item >(
 
 	return (
 		<Menu
-			align="start"
-			trigger={
-				<Button
-					size="compact"
-					className="dataviews-view-table-header-button"
-					ref={ ref }
-					variant="tertiary"
-				>
-					{ header }
-					{ view.sort && isSorted && (
-						<span aria-hidden="true">
-							{ sortArrows[ view.sort.direction ] }
-						</span>
-					) }
-				</Button>
-			}
-			style={ { minWidth: '240px' } }
+		// align="start" TODO: align prop doesn't exist on Menu — is it ok to just remove
```

Comment:
I think it's fine, I don't see a difference by removing it. I don't see such a prop in Ariakit's `Menu` API either. Maybe this was a bad copy/pasta?